src/containers/AdminDashboard/AdminController.tsx

```diff
@@ -134,7 +143,10 @@ class AdminController {
       try {
         r = await this.superagent.delete(`${process.env.BackendUrl}/book/${id}`).set('Authorization', `Bearer ${auth.token}`)
           .set('Accept', 'application/json');
-      } catch (e) { return Promise.resolve(false); }
+      } catch (e) {
+        this.warnNotif(id);
```

Comment:
You need to pass in the appropriate message if you are reusing code (which is good), so make a new param for the function and pass in 'Failed to Delete'